salt/states/disk.py

```diff
@@ -47,7 +55,12 @@ def status(name, maximum=None, minimum=None):
             ret['comment'] += 'Min must be less than max'
     if ret['comment']:
         return ret
-    cap = int(data[name]['capacity'].strip('%'))
+    minimum = int(minimum)
+    maximum = int(maximum)
+    if absolute:
+        cap = int(data[name]['available'])
```

Comment:
@cachedout, I think this should be `data[name]['used']` rather than `data[name]['available']`, because otherwise switching from percent to kilobytes will invert the meanings of `minimum` and `maximum`.  `data[name]['capacity']` reports the percent used rather than the percent available.  It is confusing because `df -P` reports the percent as 'capacity', but the number itself is the percent used.